packages/app/src/client/util/markdown-it/emoji-mart-data.ts

```diff
@@ -1,5 +1,19 @@
-import data from 'emoji-mart/data/apple.json';
 import { Emoji } from 'emoji-mart';
+import data from 'emoji-mart/data/apple.json';
+
+// Custom mapping for unavailable emoji name
+const customEmojiAliasesMapping = {
+  family_man_boy: 'man-boy',
+  family_man_girl: 'man-girl',
+  family_man_girl_girl: 'man-girl-girl',
+  family_man_girl_boy: 'man-girl-boy',
+  family_man_boy_boy: 'man-boy-boy',
+  family_woman_boy: 'woman-boy',
+  family_woman_girl: 'woman-girl',
+  family_woman_girl_girl: 'woman-girl-girl',
+  family_woman_girl_boy: 'woman-girl-boy',
+  family_woman_boy_boy: 'woman-boy-boy',
+};
```

Comment:
Sorry, If my description is not enough.

Please just fix some keywords on sandbox.md(US/JP/CN)

eg)
```
# :pencil: Emoji

See [emojione](https://www.emojione.com/)

:smiley: :smile: :laughing: :innocent: :drooling_face:

:family: :family_man_boy: :family_man_girl: :family_man_girl_girl: :family_woman_girl_girl:

:thumbsup: :thumbsdown: :open_hands: :raised_hands: :point_right:

:apple: :green_apple: :strawberry: :cake: :hamburger:

:basketball: :football: :baseball: :volleyball: :8ball:

:hearts: :broken_heart: :heartbeat: :heartpulse: :heart_decoration:

:watch: :gear: :gem: :wrench: :envelope:
```

Reply:
Emoji keywords already updated on sandbox.md(US/JP/CN)